common/logger/src/struct_log.rs

```diff
@@ -476,18 +476,14 @@ impl TCPStructLogThread {
             };
 
             // If we fail to write, exit out and create a new stream
-            if let Err(e) = stream.write(json_string.as_bytes()) {
+            if let Err(e) = stream.write_all(json_string.as_bytes()) {
                 STRUCT_LOG_SEND_ERROR_COUNT.inc();
-                println!(
+                log::error!(
```

Comment:
I think this was intentional, so errors during logging wouldn't create a positive feedback loop

Reply:
I wrote it that way, but I changed my mind and I'd rather it make the positive feedback loop, and write to text logs (maybe struct logs), than just have println! logs.